.gitignore

```diff
@@ -4,6 +4,7 @@
 /service-worker/
 /shared/
 /configurator/*.js
+!/configurator/playground-service-worker.js
```

Comment:
For my own understanding, what is configurator and how is it used when the `sandbox-base-url` is empty?

Reply:
The configurator is the site running at https://polymerlabs.github.io/playground-elements/

The `sandbox-base-url` parameter is explained best at https://github.com/PolymerLabs/playground-elements#sandbox-security